libs/language-server/src/test/validation.spec.ts

```diff
@@ -0,0 +1,94 @@
+// SPDX-FileCopyrightText: 2023 Friedrich-Alexander-Universitat Erlangen-Nurnberg
+//
+// SPDX-License-Identifier: AGPL-3.0-only
+
+import { readFileSync } from 'fs';
+import * as path from 'path';
+
+// eslint-disable-next-line @nrwl/nx/enforce-module-boundaries
+import { StdLangExtension } from '@jvalue/jayvee-extensions/std/lang';
+import { AstNode } from 'langium';
+import { NodeFileSystem } from 'langium/node';
+
+import { JayveeServices, createJayveeServices, useExtension } from '../lib';
+
+import { ValidationResult, validationHelper } from './utils';
+
+describe('jv example tests', () => {
```

Comment:
I have moved them into `@jvalue/jayvee-extensions/std/lang` because in the interpreter there will be no validation tests (and the not yet created integration test will also have no direct validation test)